templates/DO-178C/README.md

```diff
@@ -1,3 +1,35 @@
 # StrictDoc template for DO-178C standard
 
-TBD: Instructions how to use.
+This DO-178C template provides an overview of the documents required as acceptable mean of compliance.
+
+The aim of the project is also to demonstrate the use of strictdoc integrating:
+- diagrams (asciiflow)
+- images
+- bitfields
+- doxygen
+- sphinx pdf and readthedoc publication
+
+For this purpose, a toy example is used.
+
+Requirements:
```

Comment:
Please indicate that this has been tested on Linux. Later, I will extend with macOS-specific flow.